t/01-basic.t

```diff
@@ -0,0 +1,55 @@
+use strict;
+use warnings;
+
+use Test::More tests => 7;
+use FindBin qw/$Bin/;
+use lib qq{$Bin/../lib};
+
+use_ok( q{Alien::OpenMP}, q{Can load Alien::OpenMP} );
+
+# no actual constructor exists
+my $snoo = bless {}, q{Alien::OpenMP};
+
+can_ok( $snoo, qw/cflags lddlflags/ );
+
+# dev note 1: add a new lexically scoped blockfor each
+# 'ccname' that gets added to Alien::OpenMP; please
+# don't get fancy and create a loop
+
+# dev note 2: this test is not dependent on the
+# environment, see the use of a local'd version of
+# %Config::Config below for a hint on how to properly
+# add additional tests as compiler support is added to
+# this module.
+
+GCC:
```

Comment:
My suggestion was of a separate phases of info-gather and info-interpret, which would then be testable with `is_deeply`. This is still code.